Tests/Foundation/HTTPServer.swift

```diff
@@ -99,7 +99,7 @@ class _TCPSocket: CustomStringConvertible {
         listening = false
     }
 
-    init(port: UInt16?) throws {
+    init(port: UInt16?, backlog: Int32) throws {
```

Comment:
Why did you choose Int32 ?

Reply:
I propagated `backlog` parameter from `listen` function signature, and it is `Int32` there. I guess it is imported from C `int`, which is `CInt` in Swift, which, in turn, is the alias for `Int32`. 🤔 Do you think it is better to use `CInt` here instead?

Reply:
Thanks for the explanation 🙌

 No for me Int32 is the good choice, I asked only for know how did you make your choice.